TheMET/TheMET/DataModels/Department.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Department.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class Department: Decodable {
```

Comment:
The same - there is no need to make this class decodable

Reply:
Fixed